package.json

```diff
@@ -152,7 +152,7 @@
     "tslint": "^5.20.1",
     "tslint-config-standard": "^9.0.0",
     "urlencode": "^1.1.0",
-    "webdriverio": "^6.12.1"
+    "webdriverio": "^7.0.3"
```

Comment:
Any specific reason to upgrade webdriver as part of this PR. Is it part of Bot PR ?

Reply:
failing in smoke test - https://build.platform.hmcts.net/job/HMCTS_CMC/job/cmc-citizen-frontend/view/change-requests/job/PR-3014/29/console